clearnode/rpc_router_private_test.go

```diff
@@ -153,6 +153,109 @@ func TestRPCRouterHandleGetRPCHistory(t *testing.T) {
 	assert.Equal(t, uint64(1), rpcHistory[2].ReqID, "Third record should be the oldest")
 }
 
+func TestRPCRouterHandleGetUserTag(t *testing.T) {
+	// Create signers
```

Comment:
I'd be curious to see what would happen if we added the following:

```suggestion
	t.Parallel()

	// Create signers
```